package.json

```diff
@@ -4,7 +4,7 @@
   "description": "The CodeSupport Discord bot",
   "main": "./build/app.js",
   "scripts": {
-    "start": "node -r dotenv/config ./build/app.js",
+    "start": "node -r dotenv/config ./build/index.js",
```

Comment:
As I discussed with @Jason2605 in #discord-bot, `index.js` was created to separate app definition and execution. If everything is in the same place, the app is going to be executed when a test imports it, which we don't want to happen.